test/asmt-double-entry.js

```diff
@@ -60,11 +103,23 @@ describe('navigate to asmt and fill out asmts', function() {
                 .clickNested('New Assessment', done);
         });
 
+        it ('should not see AASE instrument in the instrument list', function(done) {
+            // 1995, 31563 are instrument ids for AASE instruments
+            client.isExisting('select#instrument_id option[value="1995"]').should.be.fulfilledWith(false);
+            client.isExisting('select#instrument_id option[value="31563"]').should.be.fulfilledWith(false);
+            client.call(done);
+        });
+
         it ('should select Calculation Test (instrument_id 26363)', function(done) {
             // select calculation test
             dataEntry.selectInstrument(asmtDetails.instrumentId, done);
         });
 
+        it ('should not see specialvisit option in visit dropdown list', function(done) {
+            client.isExisting('select#segment_interval option[value="specialvisit"]').should.be.fulfilledWith(false);
+            client.call(done);
+        });
```

Comment:
Do we need to test that this option doesn't exist every time that we run the test? Is it possible for the option to return?